### DISCUSSION TITLE
Embedded views [Feb 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5074] [Discord Threads]
> 🙋‍♂️ **Mark0709**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### <@832264276105625610> Shared views (& hence embedded views) are currently supported in Read Only mode. Making changes on shared views & embedded views is restricted  
#  
> 🙋‍♂️ **Mark0709**  
#### ok got it thanks for help  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1060240406769565716)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
